README.md

```diff
@@ -1,2 +1,7 @@
 # gaiden
 GetNinjas Styleguide
+
+# Quick Start
+- Clone the repository: `git clone https://github.com/getninjas/gaiden.git`
+- Configure the build links with: `npm run config`
+- Run project with: `npm run dev`
```

Comment:
@MarcoBrunoBR it's necessary to describe which port we are using (`8001` for serve `build/*` files), including the URL of docs ([http://localhost:8000/gaiden-css/index.html](http://localhost:8000/gaiden-css/index.html)).